RLTest/__main__.py

```diff
@@ -309,7 +309,12 @@ def __init__(self):
             if self.args.env.endswith('existing-env'):
                 print(Colors.Bred('can not use valgrind with existing-env'))
                 sys.exit(1)
-            vg_debugger = debuggers.Valgrind(options=self.args.vg_options, suppressions=self.args.vg_suppressions, fail_on_errors=not(self.args.vg_no_fail_on_errors), leakcheck=not(self.args.vg_no_leakcheck)
+            if self.args.vg_options is None:
+                self.args.vg_options = os.getenv('VG_OPTIONS', '--leak-check=full --errors-for-leak-kinds=definite')
```

Comment:
Why not just using `--vg-options` when running RLTest?

Reply:
Of course we can: --vg-options will continue to work exactly as before.
But sometimes, within a script, passing multiple options to vg-options gets you into bash quoting problems that make the code messy, hard to understand and maintain.
Using an env var easily resolves this.

Reply:
I do not mind also allow env var but lets not change the default, we can have the env var override the parameter and write a warnig. How does it sound?

Reply:
Ohh ok, got it.